addons/account/account_invoice.py

```diff
@@ -316,6 +317,11 @@ def _compute_payments(self):
     ]
 
     @api.model
+    def create(self, vals):
+        self = self.with_context(mail_create_nolog = True)
+        return super(account_invoice, self).create(vals)
```

Comment:
no need to assign in self. directly you can write super(account_invoice, self.with_context(mail_create_nolog = True)).create(vals)